src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -96,6 +96,18 @@ def poll_lldp_entry_updates(pubsub):
         return ret
     return data, interface, if_index
 
+def get_latest_notification(pubsub):
+    """
+    Fetches the latest notification recorded on a lldp entry.
+    """
+    latest_update_map = {}
+    while True:
+        data, interface, if_index = poll_lldp_entry_updates(pubsub)
```

Comment:
>poll_lldp_entry_updates [](http://example.com/codeflow?start=36&length=23)

In future, pubsub may return just changed fields, not all the fields of a key. So the general solution should merge set/del entries in this function, instead of picking the last one.

Reply:
Not sure if i understood your suggestion correctly, a keyspace notif doesn't contain any info about the fields changed. Eg: `__keyspace@0__:mykey del`. 

So, even if the lld_syncd logic was modified to send only one notif instead of mutiple. That'll be captured by this method. The actual logic of fetching the kv-pairs is in the downstream methods.

Reply:
Thanks for clarify. I see the flow is implemented inside this file, instead of using a library. Sorry for misleading.